client/components/seo-preview-pane/index.jsx

```diff
@@ -181,10 +181,10 @@ export class SeoPreviewPane extends PureComponent {
 						</h1>
 						<p className="seo-preview-pane__description">
 							{ translate(
-								`Below you'll find previews that ` +
-								`represent how your post will look ` +
-								`when it's found or shared across a ` +
-								`variety of networks.`
+								'Below you\'ll find previews that ' +
```

Comment:
Solved!

Problem was the [rule](http://eslint.org/docs/rules/quotes)'s autofix. Although Calypso has the `avoidEscape` flag on, the `--fix` seems to not take that into consideration if the string is using backticks.